### PR TITLE
Adds AdminTrack#update_conversion and specs.

### PR DESCRIPTION
**Pull request by ndreynolds**
Adds support for the Admin Track `UpdateConversion` operation (v4).

### CODE REVIEW

File: spec/integration/soapy_cake/admin_track_spec.rb [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-  describe '#decrypt_affiliate_link' do
-    let(:service) { :track }
-    let(:cake_method) { :decrypt_affiliate_link }
-    let(:method) { :decrypt_affiliate_link }
-    let(:cake_opts) { { a: 1 } }
-    let(:opts) { nil }
-
-    it_behaves_like 'a cake admin method'
-  end
-
```
Comment (ndreynolds) [inline spec/integration/soapy_cake/admin_track_spec.rb]: This was duplicated in `spec/lib/soapy_cake/admin_track_spec.rb` (and not really an integration spec anyway), so I took it out.

Comment (dziemba) [inline spec/integration/soapy_cake/admin_track_spec.rb]: :+1:

File: lib/soapy_cake/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SoapyCake
-  VERSION = '1.14.4'.freeze
+  VERSION = '1.15.4'.freeze
```
Comment (dziemba) [inline lib/soapy_cake/version.rb]: Shouldn't that be `1.15.0`?

**Comment by dziemba** [comment]
:mushroom:

File: lib/soapy_cake/version.rb (hunk 2 above)
Comment (ndreynolds) [inline lib/soapy_cake/version.rb]: Yep, thanks for catching that.

**Pull request merged**